### PR TITLE
Cancel istanbul gas changes

### PR DESCRIPTION
**Pull request by mrsmkl**
### Description

This removes EIP 1884 gas changes from the Istanbul hard fork.

### CODE REVIEW

File: core/vm/eips.go [hunk 1]
```diff
@@ -46,9 +45,9 @@ func EnableEIP(eipNum int, jt *JumpTable) error {
 // - Define SELFBALANCE, with cost GasFastStep (5)
 func enable1884(jt *JumpTable) {
 	// Gas cost changes
-	jt[BALANCE].constantGas = params.BalanceGasEIP1884
-	jt[EXTCODEHASH].constantGas = params.ExtcodeHashGasEIP1884
-	jt[SLOAD].constantGas = params.SloadGasEIP1884
+	// jt[BALANCE].constantGas = params.BalanceGasEIP1884
+	// jt[EXTCODEHASH].constantGas = params.ExtcodeHashGasEIP1884
```
Comment (nategraf) [inline core/vm/eips.go]: We should include these changes unless they also break things. My understanding is that they don't.

File: core/vm/eips.go [hunk 2]
```diff
@@ -46,9 +45,9 @@ func EnableEIP(eipNum int, jt *JumpTable) error {
...
+	// jt[SLOAD].constantGas = params.SloadGasEIP1884
```
Comment (nategraf) [inline core/vm/eips.go]: Adding a comment for any confused future readers.
```suggestion
	
	// Celo does not include this gas change at genesis, where the rest of EIP1884 is applied.
	// It is planned to be applied at a later date.
	// jt[SLOAD].constantGas = params.SloadGasEIP1884
```

File: core/vm/eips.go (hunk 1 above)
Comment (mrsmkl) [reply core/vm/eips.go]: Well they might break stuff similarly to how SLOAD does, but probably not in the case we detected (transfer to governance).

Comment (nategraf) [reply core/vm/eips.go]: Good call. I agree being on the safe side of this is probably better. 👌

**Pull request merged**